code/modules/cargo/goodies.dm

```diff
@@ -31,37 +31,37 @@
 /datum/supply_pack/goody/combatknives_single
 	name = "Combat Knife Single-Pack"
 	desc = "Contains one sharpened combat knive. Guaranteed to fit snugly inside any Nanotrasen-standard boot."
-	cost = 800
+	cost = 1200
 	contains = list(/obj/item/kitchen/knife/combat)
 
 /datum/supply_pack/goody/ballistic_single
 	name = "Combat Shotgun Single-Pack"
 	desc = "For when the enemy absolutely needs to be replaced with lead. Contains one Aussec-designed Combat Shotgun, and one Shotgun Bandolier."
-	cost = 3200
+	cost = 4000
```

Comment:
why increase? 3200 x 3 is more than 8k

Reply:
it's a combat shotgun. you can store 6 combat shotguns in a single case, without even having security access.